canonical/snapcraft.io
# chore: Remove legacy publisher JS

## Done
Removed the legacy publisher JS

## How to QA
- Go to https://snapcraft-io-4930.demos.haus/<SNAP_NAME>/metrics
- Make sure the page loads
- **Note**: `lint-js` will fail as this has introduced a few TS errors which will be fixed in a subsequent PR

## Testing
- [x] No testing required (explain why): Removing code

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-17013
